docs/source/user/how_to/config.rst

```diff
@@ -4,66 +4,82 @@ Configuration how-tos
 Get and set config items
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-Using the config sub-command in the |app_name| CLI, we can get configuration items like this:
-
 .. jinja:: first_ctx
-    
-    .. code-block:: console
 
-        {{ app_package_name }} config get machine
+    The configuration file is used to store settings that control the behavior, 
+    such as the machine name, the log level, and the location of the template component source files. 
+    These can also be set and retrieved using the CLI or Python API.
+
+    .. tab-set::
+
+        .. tab-item:: CLI
+
+            Using the config sub-command in the |app_name| CLI, we can get configuration items like this:
```

Comment:
I'm not clear on when to use `|app_name|` vs `{{ app_module/package_name }}`.

Reply:
The `{{ }}` format variables must be used within `.. jinja:: first_ctx` blocks, whereas `|app_name|` works anywhere. The variables that can be accessed anywhere are those defined here in the Sphinx config file: https://github.com/hpcflow/matflow-new/blob/49da0aec669e21da2950163316a6dd8a0ec377de/docs/source/conf.py#L94